google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -43,7 +43,7 @@ appengine.deployment.error.creating.staging.directory=There was an unexpected er
 appengine.generate.config.button=Generate...
 appengine.appyaml.location.label=app.yaml path\:
 appengine.dockerfile.location.browse.button=Browse for Your Dockerfile Location
-appengine.deployment.error.with.code=Deployment failed with exit code: {0}
+appengine.deployment.error.with.code=Deployment failed with exit code: {0}\nPlease make sure that you are using the latest version of the Google Cloud SDK. (See: https://cloud.google.com/sdk/)
```

Comment:
I think the answer to that question is 'no because this is Java so the \n
break in a string is platform independent '.

On Tue, Mar 29, 2016 at 4:13 PM, Etan Shaul <notifications@github.com>
wrote:

> In
> google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/556#discussion_r57791823>
> :
>
> > @@ -43,7 +43,7 @@ appengine.deployment.error.creating.staging.directory=There was an unexpected er
> >  appengine.generate.config.button=Generate...
> >  appengine.appyaml.location.label=app.yaml path\:
> >  appengine.dockerfile.location.browse.button=Browse for Your Dockerfile Location
> > -appengine.deployment.error.with.code=Deployment failed with exit code: {0}
> > +appengine.deployment.error.with.code=Deployment failed with exit code: {0}\nPlease make sure that you are using the latest version of the Google Cloud SDK. (See: https://cloud.google.com/sdk/)
>
> Since I have windows on the brain from the other PR I was just working -
> do we want to be safe here and use a platform independent line break (since
> windows typically does a crlf)?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/556/files/6dcfad7a1d95396f65d74223e3c83370c53bad3c#r57791823>
>